go/vt/vtgate/endtoend/deletetest/delete_test.go

```diff
@@ -29,13 +29,16 @@ import (
 	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
 	vttestpb "vitess.io/vitess/go/vt/proto/vttest"
 	"vitess.io/vitess/go/vt/vttest"
+
+	_ "vitess.io/vitess/go/vt/vtctl/grpcvtctlclient"
 )
 
 var (
-	cluster     *vttest.LocalCluster
-	vtParams    mysql.ConnParams
-	mysqlParams mysql.ConnParams
-	grpcAddress string
+	cluster        *vttest.LocalCluster
+	vtParams       mysql.ConnParams
+	mysqlParams    mysql.ConnParams
+	grpcAddress    string
+	tabletHostName = flag.String("tablet_hostname", "", "the tablet hostname")
```

Comment:
Allow to pass tabletHostName via args as Go does not support well name resolution in OS X using the system resolver. This is still an unsolved issue: https://github.com/golang/go/issues/12524

Reply:
Ohh this is one of the reasons it's not possible to run many of the tests on Mac OS X?